CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Bug fixes: any changes that fix issues or bug in the existing code
 - Dependencies: updates of dependency versions
 
-## [Unreleased] - yyyy-mm-dd
+## [Unreleased] - 2023-09-21
```

Comment:
Please don't change the date, it's just an example, the date will be added later when specific release section is created ;)